test/e2e/homeworkspaces/home_workspaces_test.go

```diff
@@ -80,7 +77,8 @@ func TestUserHomeWorkspaces(t *testing.T) {
 				require.EqualError(t, err, `workspaces.tenancy.kcp.dev "~" is forbidden: User "user-2" cannot create resource "workspaces" in API group "tenancy.kcp.dev" at the cluster scope: workspace access not permitted`, "user-2 should not be allowed to get his home workspace even before it exists")
 			},
 		},
-		{
+		// TODO: the following tests have to be rewritten since we don't expose workspaces via the vw server
+		/*{
 			name: "Create a workspace in the non-existing home and have it created automatically in-workspace request",
```

Comment:
remove

Reply:
all test cases or just `Create a workspace in the non-existing home and have it created automatically in-workspace request` ?